io.appflowy.AppFlowy.yml

```diff
@@ -12,6 +12,8 @@ finish-args:
   - --socket=pulseaudio
   - --share=network
   - --device=all
+  - --filesystem=home
+  - --socket=system-bus
```

Comment:
we said during the review that this permission was NOT ok. It still is not.

Reply:
I fixed it just now. Can you help to double-check the PR? https://github.com/flathub/io.appflowy.AppFlowy/pull/2